services/src/main/java/cz/muni/fi/pa165/facade/PerformanceFacadeImpl.java

```diff
@@ -0,0 +1,64 @@
+package cz.muni.fi.pa165.facade;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import cz.muni.fi.pa165.dto.CreatePerformanceDTO;
+import cz.muni.fi.pa165.dto.PerformanceDTO;
+import cz.muni.fi.pa165.entity.Hall;
+import cz.muni.fi.pa165.entity.Performance;
+import cz.muni.fi.pa165.services.BeanMappingService;
+import cz.muni.fi.pa165.services.HallService;
+import cz.muni.fi.pa165.services.PerformanceService;
+
+/**
+ *
+ * @author xtrnkal
+ */
+public class PerformanceFacadeImpl implements PerformanceFacade {
```

Comment:
A aby to bylo konzistentní, tak tady ta @Transactional asi má bejt taky.

Reply:
přidala jsem anotaci